views/index.ejs

```diff
@@ -72,6 +72,21 @@
           </div>
         </div>
       </nav>
-    <h1>Create Your VERY OWN Todolist!</h1>
+
+<div class="flex flex-col items-center justify-center h-[calc(100%-4rem)] py-2 overflow-hidden">
+  <div class="flex flex-col items-center justify-center  w-screen">
+    <h1 class="text-6xl font-bold text-white text-center">
+      Would You Rather
+    </h1>
+    <h2 class="text-2xl text-gray-100 text-center">
+      Have Fun With Your Friends And Uncover Personal Insight
+    </h2>
+    <%if (!user) { %>
```

Comment:
conditional for if user is not signed in or logged in, the page has a signin page link at the center of the page. I will put this as just a comment, but I think according to practices we are supposed to consult the changes before pushing it in.